### ISSUE TITLE
Typescript version update please

### ISSUE THREAD

**Issue by mmakrzem** [ionic-team/stencil#1212]
**Stencil version:**

```
 @stencil/core@0.15.2
```

**I'm submitting a:**

[ X] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**
npm list typescript shows:
```
`-- @stencil/core@0.15.2
  `-- typescript@2.9.2

```

**Expected behavior:**
There is a bug in the typescript definition file for event handlers in this version which appears to be fixed in typescript@3.1.4 

The latest version of typescript is currently at 3.1.6

Please update the stencil dependency to 3.1.4 or greater.

**Comment by mmakrzem** [comment]
This issue appears to be fixed in 0.16.0-4: https://github.com/ionic-team/stencil/commit/127f0d5

**Comment by manucorporat** [comment]
Stencil One updates to latest TS!
